src/lib.rs

```diff
@@ -59,7 +205,7 @@ pub use self::{
 };
 
 #[macro_use]
-pub mod macros;
+mod macros;
```

Comment:
from looking at the rendered documentation, it didn't seem like this module needed to be exported.

Reply:
Assuming the macros are still getting exported fine, it's better to have it private